docs/tools/chrome-extension/highlights.md

```diff
@@ -1,20 +1,22 @@
 ---
-id: highlights 
+id: highlights
 title: 'Highlight'
-sidebar_label: 'Highlight'
+sidebar_label: Highlight
 keywords:
   - highlights
   - posting
 ---
 
-You can access the post to highlight it from the popup window in the extension. When you clicked, it will automatically, populate the PR or issue title, here you can use our AI functions to get a summary of the highlight similar to the PR summary. 
+You can generate and post a Highlight of your PR to your OpenSauced profile with our OpenSauced.ai Chrome extension.
+
+A popup window will open when you click the extension button on your PR page on GitHub. You can navigate to "Tools" at the bottom and select "Post Highlight". It will automatically populate the PR title, and you can generate a summary of the Highlight.
```

Comment:
## Improve Paragraph

I had to improve the previous paragraph to explain how the highlight works.

And I omit the word "issue" in this section because after trying out the feature. Creating and posting a highlight with our extension only works with PR now.

Please let me know if I need to return the word "issue" if we want to reflect a new feature to include creating Highlights for issues.

**Extension on PR: It gives the "Tools" option to create highlight.**

![extension on PR](https://github.com/open-sauced/docs/assets/45172775/cb0d23d9-b527-4bd0-977b-08e37ae172b2)

**Extension on issue: There is no option to create highlight.**

![extension on issue](https://github.com/open-sauced/docs/assets/45172775/4b19226a-7a20-4f88-a0c3-91de6f86dae0)

## Suggestion

I'm not sure if this happens to everyone, but I don't have any other options than "Post Highlights" in the "Tools" menu (see the "Extension on PR" screenshot above). The existing screenshot on the page shows 4 options in "Tools" menu. We might want to update the screenshot.

Reply:
I will revert this to previous version. 👍